model/JobRole.ts

```diff
@@ -0,0 +1,5 @@
+export class JobRole {
+    id?: number
+    name?: string
+    specification?: string
```

Comment:
do these need to be optional

Reply:
The example in the week 3 notes has the properties be optional, this is to prevent any errors when creating or updating a model. The presence of the values can be checked by a validator when we move onto tickets that are focused on creating and updating.